src/WinRT.Runtime/Projections/INotifyCollectionChanged.net5.cs

```diff
@@ -35,19 +36,24 @@ public static unsafe (Action<global::System.Collections.Specialized.NotifyCollec
                     (*(delegate* unmanaged[Stdcall]<IntPtr, global::WinRT.EventRegistrationToken, int>**)ThisPtr)[7]);
             });
             return eventSource.EventActions;
-        }
-
-        public static global::System.Guid IID { get; } = new Guid(new global::System.ReadOnlySpan<byte>(new byte[] { 0xE1, 0x55, 0x01, 0x53, 0xA5, 0x28, 0x93, 0x56, 0x87, 0xCE, 0x30, 0x72, 0x4D, 0x95, 0xA0, 0x6D }));
-
-        public static IntPtr AbiToProjectionVftablePtr => INotifyCollectionChanged.Vftbl.AbiToProjectionVftablePtr;
+        }
+        public static global::System.Guid IID { get; } = new(GetIID());
+
+        private static ReadOnlySpan<byte> GetIID()
+            => FeatureSwitches.IsWuxMode
+                ? new(new byte[] { 0xd5, 0x67, 0xb1, 0x28, 0x31, 0x1a, 0x5b, 0x46, 0x9b, 0x25, 0xd5, 0xc3, 0xae, 0x68, 0x6c, 0x40 })
```

Comment:
Need to leave the `ROS<byte>` cast in all of these, otherwise we allocate an array instead of using RVA fields. Leaving this comment here but applies to all such IID initialization paths introduced in this PR, for all WUX/MUX types.